tsconfig.json

```diff
@@ -2,7 +2,7 @@
     "compilerOptions": {
         "module": "commonjs",
         "outDir": "dist",
-        "target": "es2016",
+        "target": "es2019",
```

Comment:
as officially suggested [here](https://github.com/microsoft/TypeScript/wiki/Node-Target-Mapping#node-12). Savings in `dist/` are pretty small (about 3KiB), but it's nice that async/await is no loner transpiled, yielding a pretty big readability improvement in `chrome-launcher.js` if you aren't using source maps.